### ISSUE TITLE
Question about dsvt_plain_1f_onestage.yaml on waymo

### ISSUE THREAD

**Issue by sky-fly97** [Haiyang-W/DSVT#20]
Hello, `POST_CENTER_LIMIT_RANGE: [-80, -80, -10.0, 80, 80, 10.0]` in `POST_PROCESSING`, How is this range set? Does it have an effect on performance?

**Comment by sky-fly97** [comment]
Yes, I just noticed that pcdet defaults to `[-75.2, -75.2, -2, 75.2, 75.2, 4] `and that's why I asked the question.

**Comment by Haiyang-W** [comment]
yeah, you can try it. It's all about testing. This number range may be a legacy issue with our internal version. In my view, I think it doesn't matter. :)
